trie/zkproof/writer.go

```diff
@@ -263,7 +264,16 @@ func copyAccountState(st *types.AccountWrapper) *types.AccountWrapper {
 	}
 }
 
+func isDeletedAccount(state *types.AccountWrapper) bool {
```

Comment:
How about store `xx_trace.json` into internal module?

Reply:
Those files are just stuff for unit testing, it may be too wired to separate them from the test itself ... so you mean we should put the test in some path inside internal? or they would be shared as stuff for some other tests?

Reply:
> 

Yes, i can also use that data in some test cases when add compress logic for `BlockResult struct`.

Reply:
ok, for which path they are suitable? what about `internal/utesting/blocktraces` ?

Reply:
ok!

Reply:
done